pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -174,7 +174,7 @@ func newCertRotationController(
 			Namespace:              operatorclient.OperatorNamespace,
 			Name:                   "kube-apiserver-to-kubelet-signer",
 			Validity:               1 * 365 * defaultRotationDay, // this comes from the installer
-			Refresh:                8 * 365 * defaultRotationDay, // this means we effectively do not rotate
+			Refresh:                292 * defaultRotationDay,
```

Comment:
nit: small godoc that this is 80% of the validity range and that this invariant should be maintained

Reply:
pushed, ptal.